Models/NewCard.cs

```diff
@@ -8,32 +7,36 @@ namespace APIPlugin
 	public static class NewCard
 	{
 		public static List<CardInfo> cards = new List<CardInfo>();
-		public static Dictionary<int,List<AbilityIdentifier>> abilityIds = new Dictionary<int,List<AbilityIdentifier>>();
-		public static Dictionary<int,EvolveIdentifier> evolveIds = new Dictionary<int,EvolveIdentifier>();
-		public static Dictionary<int,IceCubeIdentifier> iceCubeIds = new Dictionary<int,IceCubeIdentifier>();
-		public static Dictionary<int,TailIdentifier> tailIds = new Dictionary<int,TailIdentifier>();
 
-		public static void Add(CardInfo card, List<AbilityIdentifier> abilityIds = null, EvolveIdentifier evolveId = null,
-			IceCubeIdentifier iceCubeId = null, TailIdentifier tailId = null)
+		public static Dictionary<int, List<AbilityIdentifier>> abilityIds = new();
+		public static Dictionary<int, EvolveIdentifier> evolveIds = new();
+		public static Dictionary<int, IceCubeIdentifier> iceCubeIds = new();
+		public static Dictionary<int, TailIdentifier> tailIds = new();
+
+		public static void Add(CardInfo card, List<AbilityIdentifier> abilityIdsParam = null,
+			EvolveIdentifier evolveId = null, IceCubeIdentifier iceCubeId = null, TailIdentifier tailId = null
+		)
 		{
 			NewCard.cards.Add(card);
-			handleIdentifiers(card, abilityIds, evolveId, iceCubeId, tailId);
+			handleIdentifiers(abilityIdsParam, evolveId, iceCubeId, tailId);
 			Plugin.Log.LogInfo($"Loaded custom card {card.name}!");
 		}
 
+
 		// TODO Implement a handler for custom appearanceBehaviour - in particular custom card backs
 		// TODO Change parameter order, and function setter call order to make more sense
 		// TODO Rename parameters to be more user friendly
-		public static void Add(string name, List<CardMetaCategory> metaCategories, CardComplexity cardComplexity,
-			CardTemple temple, string displayedName, int baseAttack, int baseHealth,
-			string description = null,
-			bool hideAttackAndHealth = false, int cost = 0, int bonesCost = 0, int energyCost = 0,
+		public static void Add(string name, string displayedName, int baseAttack, int baseHealth,
+			List<CardMetaCategory> metaCategories, CardComplexity cardComplexity, CardTemple temple,
+			string description = null, bool hideAttackAndHealth = false,
+			int bloodCost = 0, int bonesCost = 0, int energyCost = 0,
 			List<GemType> gemsCost = null, SpecialStatIcon specialStatIcon = SpecialStatIcon.None,
 			List<Tribe> tribes = null, List<Trait> traits = null, List<SpecialTriggeredAbility> specialAbilities = null,
-			List<Ability> abilities = null, List<AbilityIdentifier> abilityIds = null, EvolveParams evolveParams = null,
+			List<Ability> abilities = null, List<AbilityIdentifier> abilityIdsParam = null,
+			EvolveParams evolveParams = null,
 			string defaultEvolutionName = null, TailParams tailParams = null, IceCubeParams iceCubeParams = null,
 			bool flipPortraitForStrafe = false, bool onePerDeck = false,
-			List<CardAppearanceBehaviour.Appearance> appearanceBehaviour = null, Texture2D tex = null,
```

Comment:
Wanted to make a clear distinction between the main texture and the alt texture.